src/client/extension.ts

```diff
@@ -61,9 +66,14 @@ export async function activate(
         appInsightsResource.instrumentationKey
     );
     context.subscriptions.push(_telemetry);
+    oneDSLoggerWrapper.instantiate("us");
```

Comment:
add a TODO.

// TODO: Need to decide on data residency part on unauthenticated scenario